README.md

```diff
@@ -99,6 +123,9 @@ The `--config-file` option can be used to read the configuration options from a
 ```yaml
 # required
 server: "https://example.com"
+# required for token authentication
+authToken: "example-token"
```

Comment:
change looks good to me, the only thing to consider might be changing the structure to a nested one (in case you would want to add support to client certs etc. at some point):

```
auth:
  token: "example-token"
  user: "user"
  password: "pass"
```

Reply:
I did not want to make any incompatible change in this version. I know that this is still possible technically, because we are still at 0.x, but maybe I'll do that change later before going to 1.x? WDYT?

Reply:
Sounds good!